### ISSUE TITLE
L/R balance from FIT files (again)

### ISSUE THREAD

**Issue by nbgarleanu** [GoldenCheetah/GoldenCheetah#4491]
Hi,

I export FIT files from SuperCycle. The files contain L/R balance data, but also — when viewed with Fitfileexplorer — there is a column (field "left_right_balance") whose entries (values) are all "left_right_balance_0", which may explain why GC thinks that the file contains no balance data. I don't know it to be true, but I allow for the possibility that the trailing "_0" refers to the right-bit mentioned in https://github.com/GoldenCheetah/GoldenCheetah/issues/4067.

I had thought based on that exchange that my FIT files are simply put together incorrectly, but then I uploaded them to intervals.icu, and could see the all of the balance data. The developer of SuperCycle has successfully tried other software, as well. I don't know how intervals.icu reads the FIT files, what filters it may use etc., but I presume its choice is `reasonable'.

In conclusion: can GC be instructed to recognize the balance data from such files? (Whether by default or as a user option?)

Nicolae

Issue tracker is **only** for Bugs and Features, please don't open issues for questions or technical support. Before to open a new issue please read the contributing guidelines (link below).

If you have questions, please read the FAQs and User's/Developer's Guide:
* FAQs - https://github.com/GoldenCheetah/GoldenCheetah/wiki/FAQ
* User's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/UG_Main-Page_Users-Guide
* Developer's Guide - https://github.com/GoldenCheetah/GoldenCheetah/wiki/Developers-guide

If you need help or technical support please use the forums:
* Users - https://groups.google.com/forum/#!forum/golden-cheetah-users
* Developers - https://groups.google.com/forum/#!forum/golden-cheetah-developers

**Comment by nbgarleanu** [comment]
Hi, 
I presume the issue was rejected because it was addressed elsewhere? Perhaps even in the thread I linked to? If so, is there no remark to be made concerning why there is no such problem in intervals.icu? As a user caught in the middle between two developers, I genuinely cannot ascertain what generates the imperfect compatibility, and how I might be able to use all of my power meter data in GC — but I do see that the issue does not arise with other apps than GC.
I understand that developers have tons of things to do, and can't comment on all requests, many of which undoubtedly not very thoughtful. I'd be very happy to do more reading to educate myself if there was a reasonably accessible resource.
Regards,
Nicolae
